addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4557,6 +4559,12 @@ export class OdooEditor extends EventTarget {
             this._applyCommand("insert", text);
         } else if (odooEditorHtml) {
             const fragment = parseHTML(odooEditorHtml);
+            for (const className of this.options.renderingClasses) {
+                const elements = fragment.querySelectorAll('.' + className);
+                for (const element of elements) {
+                    element.classList.remove(className);
+                }
+            }
```

Comment:
```suggestion
            const selector = this.options.renderingClasses.map(c => `.${c}`).join(',');
            for (const element of fragment.querySelectorAll(selector)) {
                element.classList.remove(...this.options.renderingClasses);
            }
```

Reply:
I have implemented the requested changes.